jenkins/shared-library/vars/defaultCiPipeline.groovy

```diff
@@ -0,0 +1,182 @@
+/* groovylint-disable NestedBlockDepth */
+import java.nio.file.Paths
+
+void call(Closure body) {
+	Map params = [:]
+	body.resolveStrategy = Closure.DELEGATE_FIRST
+	body.delegate = params
+	body()
+
+	String packageRootPath = getJenkinsfilePath()
```

Comment:
should we suppress `UnnecessaryGetter` lint ?

Reply:
ok